src/lifecycle/index.tsx

```diff
@@ -206,10 +206,11 @@ export function extractChainInfo(state: State):
     }
   | undefined {
   if (state.type == 'ConnectedState') {
+    // default is set to substrate default
```

Comment:
When is it useful? Isn't it better to gracefully handle the error, instead of relying on some default that might be wrong?

Reply:
Agreed.
When using Zombienet, somehow the values are null. Maybe something to investigate as part of the e2e tests, since this might be a blocker when we set e2e tests to run.